scene/gui/range.cpp

```diff
@@ -122,10 +129,20 @@ void Range::_set_value_no_signal(double p_val) {
 }
 
 void Range::set_value_no_signal(double p_val) {
+	bool prev_nan = is_nan_input_allowed();
+	bool prev_inf = is_infinite_input_allowed();
+
+	if (is_display_non_finite_allowed()) {
+		set_allow_non_finite(true);
+	}
+
```

Comment:
At least I think this code needs to leverage the display more

But the fact still remains that this change, without allowing users to configure it from script, means that `set_value_no_signal` called from scripting will now allow values it didn't before, which IMO is breaking